src/routing/tests/mod.rs

```diff
@@ -1279,59 +1279,6 @@ async fn handle_sync() -> Result<()> {
 
 // TODO: add test that untrusted `Sync` is not applied
 
-#[tokio::test]
```

Comment:
better to have this test refactored to show the forked chain can now be handled?